uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1589,11 +1595,9 @@ sub CreateMRICandidates {
     ## Create non-existent candidate if the profile allows for #####
     ## Candidate creation ##########################################
     ################################################################
-    if (!NeuroDB::MRI::subjectIDExists(
-            $subjectIDsref->{'CandID'},
-            $this->{dbhr}
-        ) 
-        && (NeuroDB::DBI::getConfigSetting($this->{dbhr},'createCandidates'))
+    my $configOB = $this->{'configOB'};
+    if (!NeuroDB::MRI::subjectIDExists($subjectIDsref->{'CandID'}, $this->{dbhr})
```

Comment:
No need to instantiate `$configOB` again: it was already defined on line 1585. Also, the definition of `$createCandidates` should go here instead of further up.